datasets/_global/iadb_sanctions/crawler.py

```diff
@@ -74,16 +72,12 @@ def crawl(context: Context):
             prop = "nationality" if schema == "Person" else "jurisdiction"
             entity.add(prop, country)
 
-        end_dates = h.parse_date(row.pop("to", None), FORMATS)
+        end_dates = h.parse_date(row.pop("to", None), context.dataset.dates.formats)
         is_active = False
         if end_dates:
             end_date = end_dates[0]
             today = datetime.now().isoformat()[:10]
-            if len(end_date) == 4:
-                today = today[:4]
-                context.log.warning("Only parsed year", name=title, date=end_date)
-            is_active = today < end_date
-        else:
+            is_active = today < end_date or end_date == "Ongoing"
```

Comment:
how can the parse result end_date == "Ongoing"? do you mean to check if the unparsed value is "Ongoing"?

Reply:
it turns out, contrary to my expectation, that the raw value is returned if no date or year could be parsed.